hoot-core/src/main/cpp/hoot/core/manipulators/DanglerRemover.cpp

```diff
@@ -67,8 +67,6 @@ using namespace Tgs;
 namespace hoot
 {
 
-#define SQR(x) ((x) * (x))
-
```

Comment:
Unused `#define` within a `namespace`.  Being inside of the namespace triggered the warning but it wasn't actually used anywhere.